<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>494</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>653</width>
    <height>494</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>653</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="confList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>76</width>
          <height>79</height>
         </size>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="spacing">
         <number>17</number>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <item>
         <property name="text">
          <string>General</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/images/settings/parameter.png</normaloff>:/images/settings/parameter.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/images/settings/color.png</normaloff>:/images/settings/color.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Programs</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/images/settings/program.png</normaloff>:/images/settings/program.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Network</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/images/settings/network.png</normaloff>:/images/settings/network.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="confStack">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="General">
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="generalLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>General</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="defaultGroupBox">
             <property name="title">
              <string>Default Boxes</string>
             </property>
             <widget class="QWidget" name="formLayoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>491</width>
                <height>171</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <property name="labelAlignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Number of squares:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="squareNumberBox">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="customCheck">
             <property name="text">
              <string>Disable Custom Boxes Mode</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="customGroupBox">
             <property name="title">
              <string>Custom Boxes</string>
             </property>
             <widget class="QWidget" name="formLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>491</width>
                <height>171</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="formAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="customXMLPathLabel">
                 <property name="text">
                  <string>Path:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="customXMLPathButton">
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Choose XML Folder path...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Color">
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="colorLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Color</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="verticalSpacing">
              <number>40</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Outside of boxes:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="colorOutsideCombo">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>White</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Yellow</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Orange</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Red</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Blue</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Purple</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Pink</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Green</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Brown</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Gray</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Black</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Selected box:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="colorSelectedCombo">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <property name="modelColumn">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>White</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Yellow</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Orange</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Red</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Blue</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Purple</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Pink</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Green</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Brown</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Gray</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Black</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Unselected boxes:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="colorUnselectedCombo">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>4</number>
               </property>
               <item>
                <property name="text">
                 <string>White</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Yellow</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Orange</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Red</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Blue</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Purple</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Pink</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Green</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Brown</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Gray</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Black</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Opacity (min to max):</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSlider" name="colorOpacitySlider">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>80</number>
               </property>
               <property name="tracking">
                <bool>true</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Programs">
         <widget class="QWidget" name="verticalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="programsLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Programs</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Network">
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="networkLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Network</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="proxyCheck">
               <property name="text">
                <string>Enable Proxy setting</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="proxyGroupBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Proxy setting</string>
               </property>
               <widget class="QWidget" name="formLayoutWidget_4">
                <property name="geometry">
                 <rect>
                  <x>9</x>
                  <y>19</y>
                  <width>491</width>
                  <height>371</height>
                 </rect>
                </property>
                <layout class="QFormLayout" name="formLayout_4">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                 </property>
                 <property name="labelAlignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                 <property name="formAlignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="verticalSpacing">
                  <number>40</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Hostname:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="proxyHost">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Port:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Name:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="proxyName">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Password:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="proxyPassword">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSpinBox" name="proxyPort">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>65535</number>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>customCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>customGroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>proxyGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
